src/components/Common/BlogListing.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { Padding } from 'styled-components-spacing'
+import { format } from 'date-fns'
+
+import { Grid } from '../grid'
+import TitleAndMediaList from '../Common/TitleAndMediaList'
+
+const BlogListing = ({ title, description, posts }) => {
+  const mediaItems = posts.map(
+    ({ id, title, uniqueSlug, firstPublishedAt }) => {
+      return {
+        id,
+        title,
+        href: `https://medium.com/yld-engineering-blog/${uniqueSlug}`,
+        body: format(new Date(firstPublishedAt), 'MMMM DD[,] dddd')
+      }
+    }
+  )
+
+  return (
+    <Grid>
+      <Padding vertical={{ desktop: 5, smallPhone: 3.5 }}>
+        <TitleAndMediaList
+          title={title}
+          description={description}
+          mediaItems={mediaItems}
+          CTALink="https://medium.com/yld-engineering-blog"
```

Comment:
I would store `https://medium.com/yld-engineering-blog` in a const since it's being used twice in the file.
A side note, not a requested change but CTALink / CTAText should come from Contentful.

Reply:
I see what you mean. But just because they happen to be the same, doesn't necessarily mean that they are conceptually always supposed to be identical.

For example, Greta could theoretically change the CTALink so that it directed people to the "latest stories", i.e. https://medium.com/yld-engineering-blog/latest
But we would never want the text appended to the beginning of the slug to change from https://medium.com/yld-engineering-blog

Anyway, when we do have the CTALink coming from Contentful, I think line 27 will become `CTALink={CTALink}` but line13 (the `href`) will remain hard-coded in our codebase :woman_shrugging:

Reply:
No worries. if they may not always be identical then no need to store in a const 👍